cfgov/prepaid_agreements/views.py

```diff
@@ -95,13 +95,17 @@ def get_support_text():
         title='Prepaid agreements support and inquiries').first()
 
 
+def get_valid_products():
```

Comment:
An alternative Django convention for doing this type of thing is to add this method to a [custom model manager](https://docs.djangoproject.com/en/2.2/topics/db/managers/#calling-custom-queryset-methods-from-the-manager):

```py
class PrepaidProductQuerySet(models.QuerySet):
    def valid(self):
        return self.filter(deleted_at=None)

class PrepaidProduct(models.Model):
    ...

    objects = PrepaidProductQuerySet.as_manager()
```

The cool thing about this approach is that it keeps the logic about the model with the model definition (i.e. what determines whether a PrepaidProduct is "valid") as opposed to in a separate view file. It lets you use `PrepaidProduct.objects.valid()` anywhere, like any other queryset filter, including in combinations, e.g. `PrepaidProduct.objects.valid().filter(...)`. It also makes testing easier, because instead of having to expose the separate `get_valid_products` method and import it for your test, you can just write a test against `PrepaidProduct.objects.valid()`.

Not a must-do, but maybe a nice thing to consider here or in future.

Reply:
This is definitely preferable -- thanks for the suggestion!

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5367/commits/b652e0ce0a60b6610a8f60641b7c90ad29e000e5